sources/install/package_base.sh

```diff
@@ -173,6 +173,14 @@ function install_ultimate_vimrc() {
     sh ~/.vim_runtime/install_awesome_vimrc.sh
 }
 
+function install_neovim() {
+    curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
+    chmod u+x nvim.appimage
+    ./nvim.appimage --appimage-extract
+    cp -r squashfs-root/usr/* /usr/
+    rm -rf squashfs-root
+}
```

Comment:
Please add 
- `add-to-list` in the format `tool_name,tool_url,tool_description`
- `add-test-command` with a command that's going to be executed to make sure nvim works after install. Command must return 0 if successful. For example: `command --help` usually does it.